pkg/userdata/ubuntu/userdata.go

```diff
@@ -129,7 +129,7 @@ write_files:
     set -xeuo pipefail
     mkdir -p /opt/bin /opt/cni/bin /etc/cni/net.d /var/run/kubernetes /var/lib/kubelet /etc/kubernetes/manifests /var/log/containers
     if [ ! -f /opt/bin/kubelet ]; then
-      curl -L -o /opt/bin/kubelet https://storage.googleapis.com/kubernetes-release/release/v{{ .MachineSpec.Versions.Kubelet }}/bin/linux/amd64/kubelet
+      curl -L -o /opt/bin/kubelet https://storage.googleapis.com/kubernetes-release/release/{{ .MachineSpec.Versions.Kubelet }}/bin/linux/amd64/kubelet
```

Comment:
We should parse the entered version and accept both from the user:
- `v1.9.2`
- `1.9.2`

I think using a samver library should solve that. I can do a separate pr afterwards

Reply:
issue: #36